content/courses/ada_2022_whats_new/chapters/container_aggregates.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../../global.txt
+
+Container Aggregates
+====================
+
+.. note::
+
+    Container aggregates are supported by
+
+    * GNAT Community Edition 2021
+    * GCC 11
+
+Ada 2022 introduces container aggregates. Consider this:
+
+.. code:: ada run_button project=Courses.Ada_2022_Whats_New.Container_Aggregates
+
+   pragma Ada_2022;
+
+   with Ada.Text_IO;
+   with Ada.Containers.Vectors;
+   with Ada.Containers.Ordered_Maps;
+
+   procedure Main is
+
+      package Int_Vectors is new Ada.Containers.Vectors
+        (Positive, Integer);
+
+      X : constant Int_Vectors.Vector := (1, 2, 3);
+
+      package Float_Maps is new Ada.Containers.Ordered_Maps
+        (Integer, Float);
+
+      Y : constant Float_Maps.Map := (-10 => 1.0, 0 => 2.5, 10 => 5.51);
+   begin
+      Ada.Text_IO.Put_Line (X'Image);
+      Ada.Text_IO.Put_Line (Y'Image);
+   end Main;
+
+
```

Comment:
Maybe you could add one sentence or two explaining what's the container aggregate in the example above and how it works in this context.